### ISSUE TITLE
Houdini Engine 20.5.278 for Unreal 5.4 doesn't seem to be exporting geometry from Unreal to Houdini at all

### ISSUE THREAD

**Issue by mchataway** [sideeffects/HoudiniEngineForUnreal#315]
Hi there,
Trying to use the latest version of the plugin (20.5.278) for Unreal 5.4, however no matter what I do no geometry seems to get marshaled from Unreal into Houdini.
When using Session Sync the merge nodes from the Unreal geometry are empty.
This is the case for World Outliner inputs as well as for regular Unreal asset geometry such as static meshes when I set the HDA input type to Geometry Input rather than World Input.

I've reset the plugin settings to default in Unreal.. anything else I can try?
Anyone else experiencing the same?
Thanks!
-Mark

**Comment by thedodd** [comment]
I'm seeing the same issue, specifically on my Arm Macbook. On my windows machine, it works just fine.

**Comment by RobotsMakeGames** [comment]
Yes, and we've been having the problem for months, since Houdini 20 on Unreal 5.3. It just generates no geometry with no error. There's ways around it - it seems to be fussy about something but I haven't been able to figure out what it is. Funnily enough, it wasn't a problem on my workstation, it originally started on my partner's workstation. We found that enabling session sync made it work, but that defeated the point of having HEngine - and also we only have one licence.

This was after the issues we had with world partition/OFPA and Houdini Engine creating critical errors with LinkLoader.cpp in Unreal that corrupted entire levels. Which has apparently been fixed but we got 3DS Max with Datasmith as a workaround. We don't have the same issues using HEngine in 3DS Max and Datasmith is flawless. 

And yeah. We've submitted bug reports to SideFX who told me that if there's no error there's no problem. And then talked to SideFX people on the Discord, who said that the big companies don't have the issue. Which is fine, but we don't have entire teams dedicated to maintaining their product in the pipeline. So indie issues aren't comparable to them when we're reliant on a greater extent on SideFX to get it right and not push broken stuff.

So we've bailed on Houdini Engine for Unreal. It's been nothing but problematic since H20 after years of using it. I can't see this improving. Now on a new build of the project we're encountering other errors, so it's easier at this point to expect it to be working like it did 4 years ago - not at all. Best solution is to keep SideFX stuff out of Unreal.

**Comment by mchataway** [comment]
This is still happening for me on Windows in all versions since Houdini 20.5.278
No geometry is being marshaled to Houdini from Unreal.
When using session sync, empty merge nodes are created in Houdini to represent the Unreal geometry.
In the previous working versions, the geometry was successfully marshaled to Houdini and hooked into the merge nodes.
There are no errors or any indication that anything has gone wrong, besides the fact that there is no geometry.

This is a show stopper for us and has meant we can't use Houdini Engine on our game project until we can find a fix.
Any help with ideas or ways to debug would be very much appreciated!
